platform/darwin/src/MGLAttributedExpression.h

```diff
@@ -0,0 +1,44 @@
+#import "MGLFoundation.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NSString * MGLAttributedExpressionKey NS_EXTENSIBLE_STRING_ENUM;
+
+FOUNDATION_EXTERN MGL_EXPORT MGLAttributedExpressionKey const MGLFontNamesAttribute;
+FOUNDATION_EXTERN MGL_EXPORT MGLAttributedExpressionKey const MGLFontSizeAttribute;
+
+/**
+ An `NSExpression` that has associated text formatting attibutes (such as font size or
```

Comment:
This isn’t an NSExpression. It’s something that can be inserted into a specific operator inside an NSExpression.